### ISSUE TITLE
Feature Request: Rotate Go Board

### ISSUE THREAD

**Issue by BHydden** [online-go/online-go.com#819]
### Summary

We desire the ability to rotate the go board to all 4 positions of the 90 deg. axis. to help in game review.

### Environment

Review / demo board.

### Additional Information:

As per https://forums.online-go.com/t/is-it-better-to-review-games-from-start-to-finish-or-begin-at-the-end-and-work-backwards/21865/12

**Comment by anoek** [comment]
This issue has been marked stale and will be closed soon without further activity. To keep the issue open, please respond to the comment to keep the discussion going.

**Comment by felleg** [comment]
:+1: on this feature request

**Comment by BHydden** [comment]
Yeah this would be good, reopening.

**Comment by BHydden** [comment]
@GreenAsJade you mentioned possibly wanting this for your joseki feature. Is this still something you might consider picking up?

**Comment by GreenAsJade** [comment]
It's definitely something that is worth "keeping as a request".  I'm pretty far from being able to do it right now, due to rather a long list...

**Comment by BHydden** [comment]
Yeah fair enough. Depending on how OGS maps their boards I wasn't sure if it was fairly easy (basically a matrix manipulation) or painfully complicated haha

**Comment by GreenAsJade** [comment]
I think it's probably both.  At least for me, as soon as you start throwing around maths then it's already painfully complicated  :D :D

**Comment by anoek** [comment]
We do already have some board flipping/rotation stuff that's used in the puzzles, the main complication with the feature that I see is how to present the controls in a way that it's not junking up the interface for the vast majority of people who don't care about the feature, but still handy enough for those that do.

**Comment by BHydden** [comment]
Feel like it should only be a review tool, not on standard game boards 🤔

**Comment by felleg** [comment]
Personally, I would be happy to have the ability to rotate even in standard games. When I played a physical game with a friend, we had to stop it quickly because he had to leave, so we imported it in OGS via a .sgf file we recreated from a photograph. Because we were sitting in front of each other, currently there is no way on OGS to display the board in a way that both players will recognize. 

I realize this is probably a niche usecase, but I hope you see the value it would bring to people in hybrid mode between physical and virtual games :)

**Comment by BHydden** [comment]
Yep, 100%... and if you cam find a sleek way to incorporate the function into the UI that our playerbase supports, all power to you 😉 i sincerely wish you the best of luck because ui/ux is not any of our strong suits 😅

**Comment by GreenAsJade** [comment]
We still wish we had this, let's not close it.  I wonder how we can flag it as "hey new devs, we'd love this"

**Comment by BHydden** [comment]
Maybe these flags will help 😅

**Comment by GreenAsJade** [comment]
LOL I'm not sure about "good first issue", but hey if it works :)

**Comment by TimaGribanov** [comment]
> I wonder how we can flag it as "hey new devs, we'd love this"

Here I come

**Comment by MatthewFTom** [comment]
First time I ever tried to self-review a game is when this issue came to my mind. In terms of functionality, I looked at #3280 and anoek mentioned that this might be more appropriate in Goban? Is giving it a go here appropriate or is adding it as an issue in Goban better?

**Comment by GreenAsJade** [comment]
I think you've dug up the right answer: it should be done in Goban.

Thanks for taking a look, it's been something we wish we had for a long time.
